dist/main.js

```diff
@@ -0,0 +1,119 @@
+const renderer = new Renderer()
+let board
```

Comment:
The convention is that boolean variables should start with `is`, for example: `isGameActive`